functions/README.md

```diff
@@ -0,0 +1 @@
+# Our functions will go here
```

Comment:
To assist me in putting together this functions folder, could you specify which functions exactly you want moved from your code into here?  i.e. Not sure if the connection functions will be included, or only those related to image tag state modifications and retrieval of images.

Reply:
@ackaufman will chat off-line